build-tools-internal/build.gradle

```diff
@@ -211,7 +211,7 @@ dependencies {
   api 'com.netflix.nebula:gradle-extra-configurations-plugin:5.0.1'
   api 'com.netflix.nebula:gradle-info-plugin:9.2.0'
   api 'org.apache.rat:apache-rat:0.11'
-  api "org.elasticsearch:jna:5.7.0-1"
+  api 'net.java.dev.jna:jna:5.10.0'
```

Comment:
We should actually just replace this with `versions.getProperty('jna')` to avoid having this duplicated in two places.

Reply:
Thanks, see e58285b